admin_manual/configuration_server/index.rst

```diff
@@ -22,5 +22,6 @@ ownCloud Server Configuration
    search_configuration   
    thirdparty_php_configuration
    performance_tuning
+   performance_tuning/index
```

Comment:
Yes, I see that now. I think I need to restart Monday.
Nice work and thank you.